src/lambertw.jl

```diff
@@ -0,0 +1,386 @@
+import Base: convert
+#export lambertw, lambertwbp
+using Compat
+
+const euler =
+if isdefined(Base, :MathConstants)
+    Base.MathConstants.e
+else
+    e
+end
+
+const omega_const_bf_ = Ref{BigFloat}()
+
+function __init__()
+    omega_const_bf_[] =
+        parse(BigFloat,"0.5671432904097838729999686622103555497538157871865125081351310792230457930866845666932194")
```

Comment:
Why is it necessary to rerun this every time the module is loaded? Seems like this should just be
```julia
const omega_const_bf = parse(BigFloat, "...")
```

Reply:
IIUC, some storage is malloced when the big int is constructed and Julia keeps the pointer. The next time the module runs, the memory has been freed (of course), but the address Julia stored has not changed. BANG!...  If you don't precompile, then the heap allocation is done every time the module is loaded. But, I want to precompile. AFAICT, this is the recommended way to a) allocate the storage every time the module is `used`. b) precompile the module.

Reply:
Isn't `big"0.56..."` better than `parse(...)`?